programs/server/config.xml

```diff
@@ -409,12 +409,12 @@
     <!-- Approximate size of mark cache, used in tables of MergeTree family.
          In bytes. Cache is single for server. Memory is allocated only on demand.
          You should not lower this value.
+      <mark_cache_size>5368709120</mark_cache_size>
```

Comment:
Minor: Comment style in this file is

```xml
<!-- Some_settings sometimes does something important -->
<!-- <some_setting>1</some_setting>-->
```

(i.e. the setting comment and the setting are independently commented)